src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,252 @@
+package Duke;
+
+import Duke.Exception.DataCorruptedException;
+import Duke.Exception.EmptyArgumentException;
+import Duke.Exception.WrongArgumentException;
+import Duke.Tasks.TaskType;
+import Duke.Tasks.TaskList;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.lang.reflect.Array;
+import java.util.Scanner;
+
+
+public class Duke {
+    private static TaskList list = new TaskList();
+    private static boolean toSave = true;
```

Comment:
Perhaps you may want to consider renaming the boolean in the format of `is___`, so as to adhere to the coding standard.